Remotion/Security/UnitTests/Metadata/MetadataToXmlConverterTest.cs

```diff
@@ -505,19 +505,20 @@ public void Convert_IntegrationTest ()
     [Test]
     public void ConvertAndSave_NewFile ()
     {
-      string testOutputPath = @"Core\Metadata\Testoutput";
+      var tempDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
 
-      if (!Directory.Exists(testOutputPath))
-        Directory.CreateDirectory(testOutputPath);
+      if (!Directory.Exists(tempDirectory))
+        Directory.CreateDirectory(tempDirectory);
```

Comment:
```suggestion
      Directory.CreateDirectory(tempDirectory);
```
Given the new code, the Exists-check is redundant.